CMakeLists.txt

```diff
@@ -608,6 +608,8 @@ IF (SONAME)
 	IF (LIBGIT2_FILENAME)
```

Comment:
could you have already just used this, setting `-DLIBGIT2_FILENAME=git2.dll` ?

appending to debug flags sounds like a good idea anyway. maybe check blame for why that line says cygwin.